Pipfile.lock

```diff
@@ -797,6 +790,14 @@
             "index": "pypi",
             "version": "==1.4.3"
         },
+        "appnope": {
```

Comment:
What is this for?

Reply:
It was on master when I rebased, should I remove it?

Reply:
https://pypi.org/project/appnope/ I think so